### DISCUSSION TITLE
What is “instance_id” in auth.users table?

### DISCUSSION THREAD

**Discussion by Bogdan1001** [supabase/supabase#17483] [Questions] [answered]
Hello 👋 I’ve asked my question in the title :)

**Comment by GaryAustin1** [comment, marked as answer]
It is leftover from the original gotrue source at the moment and not used by Supabase.
![image](https://github.com/supabase/supabase/assets/54564956/856900ad-0d90-4ce9-b61c-64734c56acd3)
https://github.com/orgs/supabase/discussions/1615#discussioncomment-2789248

**Reply by paulwongx** [reply]
@GaryAustin1 I understand it was originally meant to "manage users across multiple sites" and not used/to be deprecated by Supabase. 

But out of curiosity, would you have an idea on how this table was originally meant to be used? What is an example of `raw_base_config`?

Is this table used to store each website OR each website user so `raw_base_config` would have  something like ``${website_url}:${user_id}``? 

The schema for instances is below:
```sql
create table
  auth.instances (
    id uuid not null,
    uuid uuid null,
    raw_base_config text null,
    created_at timestamp with time zone null,
    updated_at timestamp with time zone null,
    constraint instances_pkey primary key (id)
  ) tablespace pg_default;
  ```

**Reply by GaryAustin1** [reply]
I have no idea.  You might check the original netlify gotrue repository.  https://github.com/netlify/gotrue
